onnx/defs/tensor/defs.cc

```diff
@@ -86,19 +86,28 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Reshape_ver5_doc = R"DOC(
+static const char* Reshape_ver12_doc = R"DOC(
 Reshape the input tensor similar to numpy.reshape.
 First input is the data tensor, second input is a shape tensor which specifies the output shape. It outputs the reshaped tensor.
 At most one dimension of the new shape can be -1. In this case, the value is
 inferred from the size of the tensor and the remaining dimensions. A dimension
 could also be 0, in which case the actual dimension value is unchanged (i.e. taken
-from the input tensor).)DOC";
+from the input tensor). If 'allowzero' is set, and the new shape includes 0, the
+dimension will be set explicitly to zero (i.e. not taken from input tensor))DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Reshape,
-    5,
+    12,
     OpSchema()
-        .SetDoc(Reshape_ver5_doc)
+        .SetDoc(Reshape_ver12_doc)
+        .Attr(
+            "allowzero",
+            "(Optional) By default, when any output dimension is equal to zero "
+            "the desired dimension value is copied from the input tensor dynamically. "
+            "allowzero=1 indicates that if any output dimension value is set to zero, "
```

Comment:
change "output dimension" to "value in the 'shape' input"

Reply:
Done!